src/API/Google/AdsAssetGroupAsset.php

```diff
@@ -121,4 +133,84 @@ public function get_assets_by_asset_group_ids( array $asset_groups_ids ): array
 		}
 
 	}
+
+	/**
+	 * Edit assets group assets.
+	 *
+	 * @param int   $asset_group_id The asset group id.
+	 * @param array $assets The assets to create.
+	 *
+	 * @return array The asset group asset operations.
+	 */
+	public function edit_operations_assets_group_assets( int $asset_group_id, array $assets ): array {
```

Comment:
💅 

What do you thing about renaming this as `edit_operations`? Otherwise we can rename  `create_operation` as `create_operation_assets_group_assets` and `delete_operation` as `delete_operation_assets_group_assets`.

In brief I was expecting some consistency in naming.

Reply:
That sounds right, I have changed it to `edit_operations`. See: [374fdf8](https://github.com/woocommerce/google-listings-and-ads/pull/1832/commits/374fdf8888da4a47c232af4e0a6a3f15273265e9)